libs/ml/include/ml/model/model.hpp

```diff
@@ -21,17 +21,39 @@
 #include "ml/dataloaders/tensor_dataloader.hpp"
 #include "ml/meta/ml_type_traits.hpp"
 #include "ml/model/model_config.hpp"
+
 #include "ml/ops/loss_functions/types.hpp"
 #include "ml/ops/metrics/categorical_accuracy.hpp"
 #include "ml/ops/metrics/types.hpp"
+
+#include "ml/ops/loss_functions/cross_entropy_loss.hpp"
+#include "ml/ops/loss_functions/mean_square_error_loss.hpp"
+#include "ml/ops/loss_functions/softmax_cross_entropy_loss.hpp"
+
 #include "ml/optimisation/optimiser.hpp"
 #include "ml/optimisation/types.hpp"
+
 #include "ml/utilities/graph_builder.hpp"
 #include "ml/utilities/graph_saver.hpp"
 
 #include <utility>
 
 namespace fetch {
+namespace ml {
+
+namespace ops {
+
+template <typename TensorType>
+class CrossEntropyLoss;
+
+template <typename TensorType>
+class MeanSquareErrorLoss;
+
+template <typename TensorType>
+class SoftmaxCrossEntropyLoss;
+}  // namespace ops
+
+}  // namespace ml
 
```

Comment:
you included loss functions headers and at the same time provided forward declarations for them. I think now of the options is enough?

Reply:
good point. done